app/code/Magento/Paypal/Controller/Express/AbstractExpress/PlaceOrder.php

```diff
@@ -49,6 +54,7 @@ public function __construct(
         \Magento\Framework\Url\Helper\Data $urlHelper,
         \Magento\Customer\Model\Url $customerUrl,
         \Magento\Checkout\Api\AgreementsValidatorInterface $agreementValidator,
+        \Magento\Sales\Model\Order\Email\Sender\InvoiceSender $invoiceSender
```

Comment:
Looks like the comma is missing here.
Also please note, the new constructor parameters should be added to the end of params list and should be optional in order to avoid backward compatibility issues.
Please check the [Contribution Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter) (section *Adding a constructor parameter*) for more details.